API/getUserById/getUserById.py

```diff
@@ -0,0 +1,80 @@
+import boto3
+from botocore.exceptions import ClientError
+import logging
+import json
+import os 
+
+dynamodb = boto3.resource('dynamodb')
+
+
+def lambda_handler(event, context):
+    """ 
+	A Lambda function used by the APIs from 100daysofcloud.com to get user details by Id
+
+	Parameters: 
+	event["queryStringParameters"]["username"] (string): 100daysofcloud user name
+  
+	Returns: 
+	object: Returns user details from 100daysofcloud account if user is valid.
+  
+	"""
+    logging.basicConfig(
+        level=logging.INFO,
+        format=f'%(asctime)s %(levelname)s %(message)s'
+    )
+
+    logger = logging.getLogger()
+
+    git_user=""
+    resp={}
+    statusCode=404
+    
+    #checking if the lambda is called by an API gateway and has query params in the call
+    if "queryStringParameters" in event:
+    	#checking if the query params from the API call has a username
+        if "username" in event["queryStringParameters"]:
+            git_user=event["queryStringParameters"]['username']
+    if git_user=="":
+        resp['message']='User Not found'
+    else:   
+        try:
+        	#checking for database name from env variables on the lambda
+            if 'databaseName' in os.environ:
+                table = dynamodb.Table(os.environ['databaseName'])
```

Comment:
We should just start at this stage. Calling the table that we need to call in a try-catch block and handle errors appropriately in the catch block

Reply:
Also, `databaseName` will for sure be in the environment variable, no need to check for it IMO. If it's not there, the function will fail anyways. If the env variable does not correlate with an actual table, it will fail anyways. Kind of unnecessary to check for this variable existing.
We gotta have unit tests that check this as well (existing vs not existing env variable)